fracatal/functional_jax.py

```diff
@@ -103,6 +140,18 @@ def lenia_update(x):
 
   return lenia_update
 
+def make_transition_function(mean, standard_deviation):
```

Comment:
Combine this with `make_update_function` with a mode argument in a future change.